tests/openvino/test_modeling.py

```diff
@@ -2482,6 +2498,9 @@ def test_compare_to_transformers(self, model_arch):
 
             additional_inputs = {"past_key_values": DynamicCache()}
 
+        if model_arch == "llama4":
+            transformers_inputs["past_key_values"] = DynamicCache()
```

Comment:
`transformers_inputs["past_key_values"] = DynamicCache()` is already performed above

Reply:
we need to reset it, becuase after inference oject will be updated